trieste/models/gpflux/sampler.py

```diff
@@ -181,7 +179,7 @@ def get_trajectory(self) -> TrajectoryFunction:
         Generate an approximate function draw (trajectory) from the deep GP model.
 
         :return: A trajectory function representing an approximate trajectory from the deep GP,
-            taking an input of shape `[N, B, D]` and returning shape `[N, B]`.
+            taking an input of shape `[N, B, D]` and returning shape `[N, B, L]`.
```

Comment:
Remember to update the descriptions of `TrajectoryFunction` and `TrajectorySampler.get_trajectory` in `models.interfaces`. (In particular, having trajectory functions support multiple outputs makes them a bit more different to acquisition functions, right?)